src/panels/lovelace/hui-root.js

```diff
@@ -50,6 +50,14 @@ class HUIRoot extends NavigateMixin(EventsMixin(PolymerElement)) {
       app-toolbar a {
         color: var(--text-primary-color, white);
       }
+      #view {
+        display: block;
+        min-height: calc(100vh - 112px);
+        position: relative;
```

Comment:
This one is only needed if we want to have things position itself absolute.

Reply:
I don't think we do. I'll take it out